code/_onclick/hud/screen_object_holder.dm

```diff
@@ -0,0 +1,59 @@
+/// A helper instance that will handle adding objects from the client's screen
+/// to easily remove from later.
+/datum/screen_object_holder
+	VAR_PRIVATE
```

Comment:
I was promised it would remain locked away, never to be shown to mortal man

Reply:
violation of our contributing standards

Reply:
who promised you that

Reply:
I use VAR_PRIVATE all the time